.gitignore

```diff
@@ -1,11 +1,12 @@
 __debug_bin
 .vscode
-*.json
```

Comment:
Warum? Verhindert dass aus versehen Prod Configs eingecheckt werden.

Reply:
Ich kann den auch spezieller machen. Konkret wurde die package[-lock].json ignoriert und gerade in der FE-Welt ist das auch nicht unüblich .json Files bspw. für i18n zu haben.

Welche Prod Configs sind das denn? Hatte im Projekt gesucht und konnte kein anderes JSON finden.

Reply:
Deshalb ja ;) Ich habe oft lokal tokens uns ähnliches liegen das nicht ins repo darf. Du kannst m.E. *.json ausschliessen und package*.json wieder erlauben?

Reply:
✅ done